### PR TITLE
enriched the documentation

### PR DESCRIPTION
**Pull request by axelearning**

### CODE REVIEW

File: docs/csm-report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# 📊 CT/GT Report Generation
+This repository provides the code needed to generate reports tracking the usage of S&D (Supply & Demand) and Globe Trotter. These reports are invaluable tools for understanding and enhancing our system's effectiveness.
```
Comment (MaximeBastion) [inline docs/csm-report.md:2]: It's `Sales & Distribution` right?

File: docs/csm-report.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+## 🎛 Configuration Explanation
+
+The following table provides a detailed explanation of the various configuration parameters used in the report:
+
+| Parameter                | Description                                                       | Possible Values                      |
+|--------------------------|-------------------------------------------------------------------|--------------------------------------|
+| `frequency`              | The frequency of report generation                               | 'daily', 'weekly', 'monthly'         |
+| `target`                 | The target value for computing productivity rate KPIs             | Any integer (default is 100)         |
+| `language`               | The language of the report                                        | 'FR' (French), 'EN' (English)        |
+| `link_geoplace`          | The geographic level assigned to field agents based on task distribution | Any valid geographic level            |
```
Comment (MaximeBastion) [inline docs/csm-report.md:17]: The name `link_geoplace` isn't super clear imo since we are talking about a geo level

**Review by MaximeBastion — CHANGES_REQUESTED** [review]

File: docs/csm-report.md (hunk 1 above)
Comment (axelearning) [reply docs/csm-report.md]: yes indeed ! Opppsi 🤖

File: docs/csm-report.md (hunk 2 above)
Comment (MaximeBastion) [reply docs/csm-report.md]: Sure, that or `linked_geo_level`

**Review by MaximeBastion — APPROVED** [review]

**Pull request merged**